knowagedao/src/main/java/it/eng/spagobi/kpi/bo/KpiValue.java

```diff
@@ -3,7 +3,7 @@
 import java.util.Date;
 
 // TODO: remove the old class it.eng.spagobi.kpi.bo.config.KpiValue
-public class KpiValue implements Cloneable {
+public final class KpiValue implements Cloneable {
```

Comment:
Why declaring the entire class to be final?

Reply:
Following the guidelines from the document: 
![image](https://github.com/KnowageLabs/Knowage-Server/assets/81435364/09eaf91e-8a49-4627-a249-8fed7809b1b4)

Because there are no clone() methods inside the class, i am declaring whole class as final.
Maybe we could remove the Cloneable implementation?